en/docs/get-started/about-this-release.md

```diff
@@ -56,6 +56,16 @@ It is now available to download from [here](https://github.com/wso2/product-apim
 - Defaulting code and configs to do with event based
 -->
 
+#### What has changed in this release
+
+##### Removed features and functionalities
+
+- Microgateway Labels
+
+      [Grouping APIs with Microgateway Labels](https://apim.docs.wso2.com/en/latest/learn/api-microgateway/grouping-apis-with-labels/)
+      is removed from this release. [Virtual Hosts (VHosts)]({{base_path}}/deploy/deploy-api/exposing-apis-via-custom-hostnames)
```

Comment:
Shall we change this as follows?

Instead, you can use [Virtual Hosts (VHosts)]({{base_path}}/deploy/deploy-api/exposing-apis-via-custom-hostnames) and dynamically manageable Gateway environments to group and expose APIs with custom hostnames.

Reply:
updated